src/reducer/wishlistReducer.jsx

```diff
@@ -0,0 +1,17 @@
+const wishlistReducer = (state, action) => {
+  switch (action.type) {
+    case "ADD_TO_WISHLIST":
+      return { ...state, wishlist: [...action.payload] };
+
+    case "REMOVE_FROM_WISHLIST":
+      return { ...state, wishlist: [...action.payload] };
```

Comment:
suggestion - same as cart reducer, here also you can use "SET_WISHLIST" beacause return values are same

Reply:
Yes...working on it